libs/coin-modules/coin-aptos/src/bridge/index.ts

```diff
@@ -48,7 +48,7 @@ function buildAccountBridge(
     createTransaction,
     updateTransaction,
     getTransactionStatus,
-    getSerializedAddressParameters,
+    getSerializedAddressParameters: account => pathToBuffer(account.freshAddressPath),
```

Comment:
why isn't bip32asBuffer in libs/coin-framework/src/bridge/jsHelpers.ts enough ?

Reply:
It doesn't return the same result:
```
pathToBuffer("44'/637'/0'/0/0"): <Buffer 05 80 00 00 2c 80 00 02 7d 80 00 00 00 80 00 00 00 80 00 00 00>
---
bip32asBuffer("44'/637'/0'/0/0"): <Buffer 05 80 00 00 2c 80 00 02 7d 80 00 00 00 00 00 00 00 00 00 00 00>
```